docs/reference/commandline/inspect.md

```diff
@@ -63,7 +63,7 @@ $ docker inspect --format='{{.LogPath}}' $INSTANCE_ID
 ### Get an instance's image name
```

Comment:
Not directly related to the PR, but I think we should call this "container" rather than "instance".